conda-recipe/meta.yaml

```diff
@@ -14,20 +14,17 @@ build:
 requirements:
   build:
     - python
-    - enaml
```

Comment:
the things in the `build` section are what `pyxrf` needs to build itself.  This does not build other libraries